### PR TITLE
feat/ccn3-898/update-validation-messages-login-journey

### PR DESCRIPTION
**Pull request by Thomas-95IS**
## Description

Updated validation messages for login journey as per ticket CCN3-898. Includes fix for validation messages during security questions.

## Todo's before PR

- [x] Rebase from develop
- [x] Unit tests passed (`make test`)
- [x] PR naming according our [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [x] PR description describes the overall goals of PR commits
- [x] Templates been checked with relevant user-story

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [x] Specified assigneses (those who'll merge)
- [x] Specified labels
- [x] Specified milestone

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [x] Code syntax formatting checked
- [x] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [x] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/forms/security_question.py [hunk 1]
```diff
@@ -42,10 +39,11 @@ class SecurityDateForm(ChildminderForms):
     GOV.UK form for the Your personal details: date of birth page
     """
     field_label_classes = 'form-label-bold'
-    error_summary_template_name = 'error-summary.html'
+    error_summary_template_name = 'standard-error-summary.html'
+    error_summary_title = 'There was a problem with your answer'
     auto_replace_widgets = True
 
-    date_of_birth = CustomSplitDateFieldDOB(label='Date of birth', help_text='For example, 31 03 1980')
+    date_of_birth = CustomSplitDateFieldDOB(label='Date of birth', help_text='For example, 31 03 1980' , required=True, error_messages={'required': 'Please give an answer'})
```
Comment (ricaduchateau) [inline application/forms/security_question.py]: Hi @Thomas-95IS (and maybe @GeevesH-IS) - is there meant to be a 'SecurityDateForm'? Is the date of birth a security question?

File: application/forms/security_question.py [hunk 2]
```diff
@@ -68,11 +66,6 @@ def clean_security_answer(self):
         birth_month = str(self.cleaned_data['date_of_birth'].month)
         birth_year = str(self.cleaned_data['date_of_birth'].year)
         if self.day != birth_day or self.month != birth_month or self.year != birth_year:
-            self.error = 'wrong'
-            raise forms.ValidationError('wrong')
-
-        if len(birth_year) == 0 or len(birth_day) == 0 or len(birth_month) == 0:
-            self.error = 'empty'
-            raise forms.ValidationError('empty')
+            raise forms.ValidationError('Your answer must match what you told us in your application')
```
Comment (ricaduchateau) [inline application/forms/security_question.py]: Hi @Thomas-95IS, is the date of birth validation meant to be part of the 'clean_security_answer' method? CC: @GeevesH-IS

File: application/forms/security_question.py (hunk 1 above)
Comment (Thomas-95IS) [reply application/forms/security_question.py]: Hi @ricaduchateau, I think it is - it's on the log in journey outlined in one of the tickets: https://jira.informed.com:8443/browse/CCN3-795

File: application/forms/security_question.py (hunk 2 above)
Comment (Thomas-95IS) [reply application/forms/security_question.py]: The `clean_security_answer()` method will be called by Django during its processing of `form.is_valid()`, so it seems sensible to me - is there anywhere else more appropriate for it to go?

File: application/forms/security_question.py (hunk 1 above)
Comment (ricaduchateau) [reply application/forms/security_question.py]: Hi @Thomas-95IS, my apologies, I didn't realise the security questions now involve the date of birth (when I last worked on Ofsted it was only a security question). I'm happy with the changes.

File: application/forms/security_question.py (hunk 2 above)
Comment (ricaduchateau) [reply application/forms/security_question.py]: Hi @Thomas-95IS, please see my previous comment; again, I wasn't aware there is now a security date of birth questions, so happy to accept this change.

Comment (Thomas-95IS) [reply application/forms/security_question.py]: No problem, thanks.

**Pull request merged**